test/test_chunkstore.py

```diff
@@ -21,16 +22,42 @@ def setUp(self) -> None:
         )
         self._store = CciChunkStore(cci_odp, dataset_id, cube_params)
 
-    @skipIf(os.environ.get('XCUBE_DISABLE_WEB_TESTS', None) == '1', 'XCUBE_DISABLE_WEB_TESTS = 1')
+    # @skipIf(os.environ.get('XCUBE_DISABLE_WEB_TESTS', None) == '1', 'XCUBE_DISABLE_WEB_TESTS = 1')
```

Comment:
Can you please comment this back in? Set the variable to 0 if you want to run this test.